src/unity/python/turicreate/toolkits/sound_classifier/sound_classifier.py

```diff
@@ -135,6 +144,8 @@ def create(dataset, target, feature, max_iterations=10, verbose=True,
         raise _ToolkitError("'%s' column is not audio data." % feature)
     if target not in dataset.column_names():
         raise _ToolkitError("Target column '%s' does not exist" % target)
+    if not isinstance(custom_layer_sizes, list) or len(custom_layer_sizes) == 0 or not isinstance(custom_layer_sizes[0], int):
```

Comment:
Nit: What if someone passes in an array of the form [75, 200, None]? We probably want a check for all elements in the array?

We may want to add unit tests for corner cases (but maybe rarely encountered cases) like these?

Reply:
Good points. I added more checks here and manually tested the cases you mentioned.